spec/models/order_spec.rb

```diff
@@ -54,70 +54,55 @@ def define_purchasable_instrument
     let(:user) { create(:user) }
 
     context "actual" do
-      before :each do
-        @cost = @subsidy = 0
-
-        (1..4).each do |i|
-          cost = 10 * i
-          subsidy = 5 * i
-          @cost += cost
-          @subsidy += subsidy
-          order.order_details.create(attributes_for(:order_detail,
-                                                    product_id: item.id,
-                                                    account_id: account.id,
-                                                    actual_cost: cost,
-                                                    actual_subsidy: subsidy,
-                                                    price_policy_id: price_policy.id,
-                                                   ))
-        end
-
-        @total = @cost - @subsidy
-      end
+      let!(:order_detail_1) { create(:order_detail, order: order, product: item, account: account, actual_cost: 10, actual_subsidy: 5, price_policy: price_policy) }
+      let!(:order_detail_2) { create(:order_detail, order: order, product: item, account: account, actual_cost: 12, actual_subsidy: 0, price_policy: price_policy) }
+      let(:cost) { 22 }
```

Comment:
If they're only used once, I would just put these inline in the examples.

Reply:
Looks like you put the other expected values inline. Do we want to put these inline as well?